mitodl/mit-open
# Add custom label for percolate query subscriptions

### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/5598

### Description (What does it do?)
This PR adds a "display label" field to percolate queries which overrides any other label for display on the frontend

### Screenshots (if appropriate):
<img width="947" alt="Screenshot 2024-09-25 at 3 36 49 PM" src="https://github.com/user-attachments/assets/a17fd85d-658f-443c-a192-7cc52db85ddc">

### How can this be tested?
1. Checkout this branch and run migrations.
2. Subscribe/follow to a few things
3. Go to [django admin](http://open.odl.local:8063/admin/learning_resources_search/percolatequery/) and edit a percolate query you are subscribed to. Put something custom in the "Display label" field
4. go to your settings page and view your subscriptions list. note that the subscription displays your custom label
